boards/arm/adafruit_itsybitsy_nrf52840/doc/index.rst

```diff
@@ -0,0 +1,164 @@
+.. _adafruit_itsybitsy_nrf52840:
+
+Adafruit ItsyBitsy nRF52840
+###########################
+
+Overview
+********
+
+The Adafruit ItsyBitsy nRF52840 Express is a small (36 mm x 18 mm) ARM
+development board with an onboard RGB LED, USB port, 2 MB of QSPI flash,
+and range of I/O broken out onto 21 GPIO pins.
+
+.. image:: img/adafruit_itsybitsy_nrf52840.jpeg
+     :align: center
+     :alt: Adafruit ItsyBitsy nRF52840 Express
+
+Hardware
+********
+- nRF52840 ARM Cortex-M4F CPU at 64MHz
+- 1 MB of flash memory and 256 KB of SRAM
+- 2 MB of QSPI flash
+- A user LED
+- A user switch
+- An RGB DotStar LED
+- Native USB port
+- One reset button
+
+Supported Features
+==================
+
+The Adafruit ItsyBitsy nRF52840 board configuration supports the
+following hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C       | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| SPI       | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features have not been enabled yet for this board.
+
+Connections and IOs
+===================
+
+The `Adafruit ItsyBitsy nRF52840 Express Learn site`_ has detailed
+information about the board including `pinouts`_ and the `schematic`_.
+
+LED
+---
+
+* LED0 (red) = P0.06
+
+* LED1 (Adafruit DotStar)
+
+    * DATA = P0.08
+    
+    * CLK = P1.09
```

Comment:
```suggestion
* LED0 (red) = P0.06
* LED1 (Adafruit DotStar)

    * DATA = P0.08
    * CLK = P1.09
```
to solve the compliance error reported for line 83 and to get rid of extra blank lines.